api/swagger.json

```
@@ -0,0 +1,1304 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "This is the API specification for Chapter",
+    "version": "1.0.0",
+    "title": "Chapter",
+    "termsOfService": "http://chapter.io/terms",
```

Use FCC's terms for now
```suggestion
    "termsOfService": "https://www.freecodecamp.org/news/terms-of-service/",
```